querybook/server/const/metastore.py

```diff
@@ -4,3 +4,29 @@
 class DataTableWarningSeverity(Enum):
     WARNING = 0
     ERROR = 1
+
+
+class MetadataType(Enum):
+    TABLE_DESCRIPTION = "table_description"
+    COLUMN_DESCRIPTION = "column_description"
+    OWNER = "owner"
+    TAG = "tag"
+    DOMAIN = "domain"
+
+
+class MetadataMode(Enum):
+    READ_ONLY = "read_only"  # metadata will be read-only on Querybook UI
```

Comment:
My thought was: if a metadata type key is not in the loader config, then it will be editable and only saved in querybook db, which is today's behavior.

I can also add another option for it, and make it default
read_only: 
write_back:  default
write_through: write to querybook db and also write back to metastore